HW3 block6/block6.css

```
@@ -85,14 +95,46 @@
   text-align: center;
 }
 
-.python-img{
-  float: left;
+.python-certified:hover, .python-certified:focus, .python-certified:active{
```

Please change format 
```suggestion
.python-certified:hover, 
.python-certified:focus, 
.python-certified:active {
```